website/docs/tutorials/guides/testing.md

```diff
@@ -825,6 +825,75 @@ There is no type checking for `variable_name`, `key`, `variable_type` make sure
 `key` is a list of arguments because cairo `@storage_var` maps any number of felt arguments to any number of felt values
 :::
 
+### `reflect`
+```python
+def reflect(self, value: Union[VmConstsReference, RelocatableValue, int]) -> Union[NamedTuple, RelocatableValue, int]:
+```
+Converts Cairo object into Python `NamedTuple` (complex structure) or keeps it a simple type `RelocatableValue` (pointer) or `int` (felt). It can be used to easily print and compare complex structures.
```

Comment:
I would rather keep the fact that we use `NamedTuple` as underlying implementation basis an implementation detail -- not a documented feature.

I believe we should leave us a way to use other means of implementation if we happen to find out that `NamedTuple` is a limiting factor.

Also, I would try to prevent tuple unpacking from being possible to write in users' code. It doesn't make any sense actually, IMHO. Namely, this hint should raise an Error:

```cairo
%{ a, b, c = reflect(ids).foobar %}
```